content/using/install.md

```diff
@@ -6,37 +6,33 @@ template = "page_indiced.html"
 aliases = ["/docs/getting-started/"]
 +++
 
-This guide covers how to set up and install your own Urbit. Running Urbit on your own is somewhat technical, so if you're not technically inclined you may want to consider a [hosting provider](#hosting-providers).
+This guide covers how to install and set up your own planet, but installation for a star is exactly the same. You will first install the Urbit binary, then you will either boot a comet or a planet.  It's not too difficult, but if you would prefer to have someone run your urbit for you, consider a [hosting provider](#hosting-providers).
 
-## Overview 
 
-There are two major components of the Urbit ecosystem: **Urbit OS** and **Urbit ID**.
-
-The Urbit peer-to-peer (P2P) network is composed of instances of Urbit OS — also called **ships** — that run as virtual machines on any Unix platform. The Urbit ID registry is deployed on the Ethereum blockchain, and individual IDs exist as non-fungible ([ERC-721](https://eips.ethereum.org/EIPS/eip-721)) tokens. On the network, Urbit IDs function as network addresses, represented by pronounceable names like `~padmyn-pasnux`.
-
-As we'll explain, there are a few types of Urbit ID, but everyday users tend to use **planets**, as full citizens of the network.
+## Installing Urbit
 
-Two other types of ships are **stars** and **galaxies**. Together these perform essential infrastructural roles for the Urbit network. Each galaxy can distribute 256 stars, and each star can distribute 65,536 planets. So any given planet ultimately came from a specific star, which itself came from a specific galaxy. On the Urbit OS side, stars help route packets; galaxies are more like DNS root servers or ICANN members. The difference with Urbit in the analogy is that Urbit IDs are owned cryptographically by individuals, collectives, and companies alike, and accrue reputation independently.
+You'll want the Urbit binary installed first.
 
-If you just want to try Urbit out, there are also free IDs called **comets**. They are good for bots and people who are new to Urbit and want to check out the network, but they aren't good for building a reputation as a friendly and responsible peer. Their long names make them difficult to remember—and some communities ban comets to prevent spam—but they're still a good way to see what Urbit is about before buying your own planet.
+### Windows
```

Comment:
I don't think Windows should be listed first. The order should be MacOS, then Linux, then Windows.

Reply:
I agree with you in principle, but practically, the windows instructions have to be done first and then the user moves on to the unix instructions, so to preserve the "forward only" goal of the docs, I put Windows first.

Reply:
I think this is a formatting issue. At some point we should have three tabs

Reply:
So maybe MacOS, then Windows, then Linux?

Reply:
I don't see any real difference here.  Does anyone else feel strongly?

Reply:
While leading with Windows is aesthetically unappealing, I don't think it presents any major problems.

Reply:
Updated to reference M1 macs.  

I'm inclined to leave the order as it is.